sdks/java/core/src/main/java/org/apache/beam/sdk/Pipeline.java

```diff
@@ -284,19 +274,16 @@ public void visitPrimitiveTransform(Node node) {
   }
 
   /**
-   * Runs this {@link Pipeline} using the default {@link PipelineOptions} provided
-   * to {@link #create(PipelineOptions)}.
-   *
-   * <p>It is an error to call this method if the pipeline was created without
-   * a default set of options.
+   * Runs this {@link Pipeline} according to the {@link PipelineOptions} used to create the {@link
+   * Pipeline} via {@link #create(PipelineOptions)}.
    */
   public PipelineResult run() {
     return run(defaultOptions);
   }
 
   /**
-   * Runs this {@link Pipeline} using the given {@link PipelineOptions}, using the runner
-   * specified by the options.
+   * Runs this {@link Pipeline} using the given {@link PipelineOptions}, using the runner specified
+   * by the options.
```

Comment:
Note that this method should generally be assumed to be asynchronous, and reference `PipelineResult#waitUntilFinish()`